_operator_reference/artifact.md

```diff
@@ -0,0 +1,275 @@
+
+
+## GitHub
+
+**spec.spinnakerConfig.config.artifacts.github**
+
+```yaml
+github:
+  accounts:
+  - name: my-github
+    username:
+    password:
+    usernamePasswordFile:
+    token:
+    tokenFile:
+  enabled: true
+```
+
+- `enabled`: true or false
+
+### Account parameters
+
+ - `username`: GitHub username
+ - `password`: GitHub password
+ - `usernamePasswordFile`: File containing "username:password" to use for GitHub authentication
+ - `token`: GitHub token
+ - `tokenFile`: File containing a GitHub authentication token
+
+## GitLab
+
+**spec.spinnakerConfig.config.artifacts.gitlab**
+
+```yaml
+gitlab:
+  enabled: false
+  accounts:
+  - name: gitlab-account
+    token:
+    tokenFile:
+```
+
+- `enabled`: true or false
+
+### Account parameters
+
+ - `token`: Gitlab token
+ - `tokenFile`: File containing a Gitlab authentication token
+
+## GitRepo
+
+**spec.spinnakerConfig.config.artifacts.gitrepo**
+
+```yaml
+gitrepo:
+  enabled: false
+  accounts:
+  - name:
+    username:
+    password:
+    usernamePasswordFile:
+    token:
+    tokenFile:
+    sshPrivateKeyFilePath:
+    sshPrivateKeyPassphrase:
+    sshKnownHostsFilePath:
+    sshTrustUnknownHosts:
+```
+
+- `enabled`: true or false
+
+### Account parameters
+
+- `username`: Git username
+- `password`: Git password
+- `usernamePasswordFile`: File containing "username:password" to use for Git authentication
+- `token`:  Git token
+- `tokenFile`: File containing a Git authentication token
+- `sshPrivateKeyFilePath`: Path to the ssh private key in PEM format
+- `sshPrivateKeyPassphrase`: Passphrase for encrypted private key
+- `sshKnownHostsFilePath`: File containing the known and trusted SSH hosts.
+- `sshTrustUnknownHosts`: Setting this to true allows Spinnaker to authenticate with unknown hosts
+
+## Helm
+
+**spec.spinnakerConfig.config.artifacts.helm**
+
+```yaml
+helm:
+  enabled: false
+  accounts:
+  - name: my-helm-account
+    repository:
+    username:
+    password:
+    usernamePasswordFile:
+```
+
+- `enabled`: true or false
+#
+## Account parameters
+
+ - `repository`: Helm chart repository
+ - `username`: Helm chart repository basic auth username
+ - `password`: Helm chart repository basic auth password
+ - `usernamePasswordFile`: File containing "username:password" to use for helm chart repository basic auth
+
+
+## HTTPS
+
+**spec.spinnakerConfig.config.artifacts.https**
+
+```yaml
+http:
+  enabled: false
+  accounts:
+  - name: my-http-account
+    username:
+    password:
+    usernamePasswordFile:
+```
+
+- `enabled`: true or false
+
+### Account parameters
+
+ - `username`: HTTP basic auth username
+ - `password`: HTTP basic auth password
+ - `usernamePasswordFile`: File containing "username:password" to use for HTTP basic auth
+
+
+## Maven
+
+ **spec.spinnakerConfig.config.artifacts.maven.accounts**
+
+ ```yaml
+ maven:
+   enabled: false
+   accounts:
+   - name: my-maven-account
+ 	repositoryUrl:
```

Comment:
There's a funky black line here. Might be cause of the change from spaces to tab? https://s.armory.io/d5u0KJlD

Reply:
I didn't see that in mine, but deleted and spaced anyway